spring-cloud-app-broker-deployer-cloudfoundry/src/test/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployerUpdateApplicationTest.java

```diff
@@ -142,6 +153,7 @@ void setUp() {
 		given(operationsApplications.pushManifest(any())).willReturn(Mono.empty());
 		given(resourceLoader.getResource(APP_PATH)).willReturn(new FileSystemResource(APP_PATH));
 
+		given(cloudFoundryOperations.services()).willReturn(operationsServices);
```

Comment:
This mock was for creating services on update, so we probably don't need it in 1.0.x.

Reply:
thanks. I'll remove it.